src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -143,14 +153,14 @@ private int AnalyzeTemplate(FileInfo templateFilePath, FileInfo parametersFilePa
                 if (!templateFilePath.Exists)
                 {
                     logger.LogError("Invalid template file path: {templateFilePath}", templateFilePath);
-                    return 2;
+                    return (int)ExitCode.ErrorInvalidPath;
                 }
 
                 // Check that output file path provided for sarif report
                 if (writer == null && reportFormat == ReportFormat.Sarif && outputFilePath == null)
                 {
                     logger.LogError("Output file path was not provided.");
```

Comment:
This isn't part of the scope of your PR, but I want to recommend a different wording for this error message.

Reply:
I'm listening